.github/workflows/draft-release.yml

```diff
@@ -24,7 +24,7 @@ jobs:
             exit 1
           fi
           echo "LIFECYCLE_VERSION=$version" >> $GITHUB_ENV
-      - name: Determine download urls for linux-x86-64, linux-arm64, and windows
+      - name: Determine download urls for linux-x86-64, linux-arm64, linux-s390x, and windows
```

Comment:
We should also expect 10 artifacts below (we recently removed cosign key and sha, which is why the number changed)

Reply:
I changed it to 10.